docs/references/general/grid.md

```diff
@@ -0,0 +1,104 @@
+
+| Field name  | Units           | Dimensionality | Data Type   | Description                                                                 |
+| :---------- | :-----:         | :-------:      | :---:       | :---                                                                        |
+| *stid*      |  **None**       |  *[numstid]*     | ***short*** | A list of of station IDs that provided data for the record |
+| *channel*   |  **None**       |  *[numstid]*     | ***short*** | A list of channel numbers associated to the station id the record |
+| *nvec*      | **None** | *[numstid]*  | ***short*** | Number of vectors for each station|
+| *freq*      | *kHz* | *[numstid]* | ***float*** | Transmitted frequency for each radar |
+| *major.revision* | **None** | *[numstid]* | ***short*** | Major `make_grid` version number per station number |
+| *minor.revision* | **None** | *[numstid]* | ***short*** | Minor `make_grid` version number per station number |
+| *program.id*     | **None** | *[numstid]* | ***short*** | Control program ID per station number 
+| *noise.mean*      | **None** | *[numstid]* | ***float*** | Mean noise for each radar |
+| *noise.sd*        | **None** | *[numstid]* | ***float*** | Noise Standard deviation per station number |
+| *gsct*            | **None** | *[numstid]* | ***short*** | A list of groundscatter flags associated to each radars data for the given record |
+| *v.min*           | *m/s* | *[numstid]* | ***float*** | Velocity minimum threshold per station number |
+| *v.max*           | *m/s* | *[numstid]* | ***float*** | Velocity maximum threshold per station number |
+| *p.min*           | **None** | *[numstid]* | ***float*** | Power minimum threshold per station number |
+| *p.max*           | **None** | *[numstid]* | ***float*** | Power maximum threshold per station number |
+| *w.min*           | *m/s*  | *[numstid]* | ***float*** | Spectral width minimum threshold per station number |
+| *w.max*           | *m/s*  | *[numstid]* | ***float*** | Spectral width maximum threshold per station number |
+| *ve.min*          | *m/s*  | *[numstid]* | ***float*** | Velocity error minimum threshold per station number |
+| *ve.max*          | *m/s*  | *[numstid]* | ***float*** | Velocity error maximum threshold per station number |
+| *vector.mlat*     | *degrees* | *[numv]* | ***float*** | Magnetic Latitude per vector point |
+| *vector.mlon*     | *degrees* | *[numv]* | ***float*** | Magnetic Longitude  per vector point |
+| *vector.kvec*     | *degrees*  | *[numv]*   | ***flost*** | Magnetic Azimuth per vector point |
+| *vector.stid*     | **None**  | *[numv]*   | ***short*** | Station identifier per vector point|
+| *vector.channel*  | **None**  | *[numv]*   | ***short*** | Channel number per vector point |  
+| *vector.index*    | **None**  | *[numv]*   | ***int***   | Grid indices|
+| *vector.vel.median* | *m/s* | *[numv]*   | ***float*** | Velocity medians per vector point|
```

Comment:
@egthomas Haha!! Thanks for noticing (and knowing) that :smile: 
Does this also apply to other similarly-named fields such as `vector.wdt.median`?

Reply:
Yes, that's right.  Unfortunately that information (at least until this pull request is approved) is currently only found in the comments I added to the source code a few years ago:  https://github.com/SuperDARN/rst/blob/master/codebase/superdarn/src.lib/tk/gtable.2.0/src/gtable.c#L169-L182

Reply:
```suggestion
| *vector.vel.median* | *m/s* | *[numv]*   | ***float*** | Weighted mean velocity per vector point|
```
Adding in the per vector point notation here.

Reply:
I still feel that including "per vector point" (or station) on each row is necessary, as it clutters the table and is already described in the Dimensionality column.  Particularly since a "point" refers to a location, while there can be multiple grid vector measurements per grid cell.

Reply:
Agree with @egthomas here.

Reply:
OK, no strong feelings here and we can come back to this if it's not so clear. I'll edit my other notations.